src/deploy/functions/release/index.ts

```diff
@@ -105,12 +106,8 @@ export async function release(
   printTriggerUrls(wantBackend);
 
   const haveEndpoints = backend.allEndpoints(wantBackend);
-  const deletedEndpoints = Object.values(plan)
-    .map((r) => r.endpointsToDelete)
-    .reduce(reduceFlat, []);
-  if (experiments.isEnabled("automaticallydeletegcfartifacts")) {
-    await containerCleaner.cleanupBuildImages(haveEndpoints, deletedEndpoints);
-  }
+
+  await checkArtifactCleanupPolicies(options.projectId!, haveEndpoints);
```

Comment:
Won't this leak if a customer had a lone function in a region and they cleaned it up before we could check policies?